experimental/design-notes.txt

```diff
@@ -0,0 +1,112 @@
+I started comparing OpenTelemetry metrics to other metric APIs and wanted to jot down observations and questions that came up along the way:
+
+Micrometer (MM) - https://micrometer.io/docs/concepts
+
+1. Do the names that OpenTelemetry chose match industry conventions? 
+For the MM library there are different types of Meters such as timers, counters, gauges, etc. Depending on their type, meters implement operations such
+as increment() and set(). Meters are associated with a single set of tags.
+For the OT spec Meters are factories that create instruments and it is the instruments that have the data collecting APIs such as Add() or Record(). Instruments
+aren't associated with a specific set of tags (labels) unless they are first bound.
+As best I can tell the best conceptual mapping is that MM's Meter == OT's bound instrument. It doesn't worry me too much that OT would use a different
+name for the same concept, what worries me more is if OT is reusing the same names but altering the concept they apply to.
```

Comment:
Since Micrometer is Java specific, there might be less concern.
We can check what's the convention in Prometheus as it is well established and it is cross-langages.

Reply:
added "eamples-other-apis.md" document to show examples of other API patterns.